lib/api/controllers/funnelController.js

```diff
@@ -19,7 +19,7 @@ var
   ServiceUnavailableError = require('kuzzle-common-objects').errors.ServiceUnavailableError,
   Request = require('kuzzle-common-objects').Request;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Define this declaration in a local scope or bind explicitly the property to the global object. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3AS3798)